autoPlay.user.js

```diff
@@ -203,7 +204,7 @@ function MainLoop() {
 		useGoldRainIfRelevant();
 		useMetalDetectorIfRelevant();
 		useCrippleMonsterIfRelevant();
-
+		AutoUseMedicIfRelevant()
```

Comment:
The function name should be changed to "useMedicIfRelevant" to be consistent with the other functions.
You also forgot a semicolon here (welcome to JS :P).